### PR TITLE
Qgep swmm phase3 enhancements

### PR DESCRIPTION
**Pull request by tproduit**
* Reorganise QGEP SWMM doc
* Update doc according to QGEP SWMM phase 3

### CODE REVIEW

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 1]
```diff
@@ -93,3 +93,22 @@ Missing population densities
 ``qgep_od.catchment_area.population_density_current`` or ``qgep_od.catchment_area.population_density_planned`` are not filled.
 
 
+Subcatchment and rain Gages
```
Comment (sjib) [inline en/qgep_swmm/Check-SWMM-File.rst:96]: Subcatchment and Rain gauges

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 2]
```diff
@@ -93,3 +93,22 @@ Missing population densities
...
+---------------------------
+If you want to attach every subcatchments to a single rain gage, edit the file with SWMM interface:
```
Comment (sjib) [inline en/qgep_swmm/Check-SWMM-File.rst:98]: If you want to attach every subcatchments to a single rain gauge, edit the file with SWMM interface:

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 3]
```diff
@@ -93,3 +93,22 @@ Missing population densities
...
+
+- Edit > Select All
+- Edit > Group Deletion > All Rain Gages
+- Create a new raingage
+- Edit > Select All
+- Edit > Group Edit
+
+  - For objects of type: Subcatchment
+	- Edit the property: Rain Gage
```
Comment (sjib) [inline en/qgep_swmm/Check-SWMM-File.rst:107]: - Edit the property: Rain Gauge

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 4]
```diff
@@ -93,3 +93,22 @@ Missing population densities
...
+  - By replacing it with: Name of the new rain gage
```
Comment (sjib) [inline en/qgep_swmm/Check-SWMM-File.rst:108]: - By replacing it with: Name of the new rain gauge

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 5]
```diff
@@ -93,3 +93,22 @@ Missing population densities
...
+
+To edit the rain serie:
+
+- Click on Time Series
+- Double click default_qgep_raingage_timeserie
```
Comment (sjib) [inline en/qgep_swmm/Check-SWMM-File.rst:113]: Double click default_qgep_raingauge_timeserie

File: en/qgep_swmm/Check-SWMM-File.rst (hunk 1 above)
Comment (sjib) [reply en/qgep_swmm/Check-SWMM-File.rst:96]: I think it is rain gauges - not rain gages - see https://en.wikipedia.org/wiki/Rain_gauge

File: en/qgep_swmm/Check-SWMM-File.rst [hunk 6]
```diff
@@ -93,3 +93,22 @@ Missing population densities
 ``qgep_od.catchment_area.population_density_current`` or ``qgep_od.catchment_area.population_density_planned`` are not filled.
 
 
+Subcatchment and rain Gages
+---------------------------
+If you want to attach every subcatchments to a single rain gage, edit the file with SWMM interface:
+
+- Edit > Select All
+- Edit > Group Deletion > All Rain Gages
```
[the remark this answers is not in the document]
Comment (tproduit) [reply en/qgep_swmm/Check-SWMM-File.rst:101]: In swmm interface, "rain gages" is used. I will not change this one.

File: en/qgep_swmm/Check-SWMM-File.rst (hunk 3 above)
Comment (tproduit) [reply en/qgep_swmm/Check-SWMM-File.rst:107]: Same here, I will let the same spelling as in the interface

**Comment by ponceta** [comment]
@tproduit could you please check and solve conflicts to merge this?

**Comment by sjib** [comment]
@tproduit Does this need updating with the latest corrections?
https://github.com/QGEP/datamodel/pull/214

**Comment by tproduit** [comment]
> @tproduit Does this need updating with the latest corrections? [QGEP/datamodel#214](https://github.com/QGEP/datamodel/pull/214)

No, no need to update the documentation

**Comment by sjib** [comment]
@tproduit Thanks for this enhancement

**Pull request merged**